modules/hdinsight-storage/outputs.tf

```diff
@@ -3,6 +3,11 @@ output "storage_container_id" {
   description = "ID of the azure storage container"
 }
 
+output "storage_container_name" {
+  value       = azurerm_storage_container.storage_container.name
```

Comment:
A pattern we could try to implement more is to return the entire object instead of dictating which fields we choose to expose. Would give the user of the module the agency to use whatever fields they need.

Pulling out the really useful things explicitly (names, IDs, addresses) for our expected use is something we should continue to do.